src/redux/categories/categories.js

```diff
@@ -1,7 +1,12 @@
 // Action
 const CHECK_STATUS = 'bookstore/categories/CHECK_STATUS';
 
-export const category = [];
+export const checkStatus = (payload) => ({
+  type: CHECK_STATUS,
+  payload,
```

Comment:
[OPTIONAL]

- Since the action you are dispatching is not receiving any payload, I think it would be better if you don't add the payload so that any other person checking or maintaining your code will not be confused 👍🏼 .